packages/kit/src/runtime/client/client.js

```diff
@@ -318,7 +318,6 @@ async function _invalidate() {
 		}
 	}
 
-	invalidated.length = 0;
```

Comment:
This would probably bring back #10677?

Reply:
I see tests failing in the PR but can't find a useful error

Reply:
It looks like #11268 did not fix that issue completely. Walking through the instructions in https://github.com/davideaster/sveltekit-invalidations I can still reproduce the error in the last step on the latest commit of sveltekit.

However, when I check out this PR, I can confirm that it brings back the first error from the reproduction repo I've linked above. The PR fixed that first error (`invalidate on a non-depends resource following invalidate with depends`).